dashboard/src/auth/Login.js

```diff
@@ -12,6 +13,7 @@ const LoginView = ({username, password, onUsernameChanged, onPasswordChanged, do
         <label>Password</label>
         <input type="password" value={password} onChange={(e) => onPasswordChanged(e.target.value)}/>
       </Form.Field>
+      <Form.Button className={css`display:none`} type="submit" />
```

Comment:
Any particular reason you have a hidden submit button here? Pressing enter in the input field should already trigger the form submit.

Reply:
Also curious why you're using `css` here but `styled` elsewhere as most uses of `styled` can also be replaced with `css` (and generally vice versa unless you want to apply the same class to different components, in which case you'll always need `css`).

Reply:
hidden button was out of need. I expected it to work without, but it did not.

Reply:
I like `styled` better, but that did not work in combination with `Form.Button`.